rules/cross-platform/threat_intel_module_match.toml

```diff
@@ -0,0 +1,144 @@
+[metadata]
+creation_date = "2021/04/21"
+maturity = "production"
+updated_date = "2021/04/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule is triggered when indicators from the Threat Intel Filebeat module has a match against local file or network observations.
+"""
+from = "now-10m"
+index = ["auditbeat-*", "endgame-*", "filebeat-*", "logs-*", "packetbeat-*", "winlogbeat-*"]
+interval = "9m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Threat Intel Filebeat Module Indicator Match"
+note = """ ## Triage and Analysis
+If an indicator matches a local observation, the following enriched fields will be generated to identify the indicator, field, and type matched.
+
+- `threatintel.indicator.matched.atomic` - this identifies the atomic indicator that matched the local observation
+- `threatintel.indicator.matched.field` - this identifies the indicator field that matched the local observation
+- `threatintel.indicator.matched.type` - this identifies the indicator type that matched the local observation
+"""
+references = [ "https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html"]
+risk_score = 99
+rule_id = "dc672cb7-d5df-4d1f-a6d7-0841b1caafb9"
+severity = "critical"
+tags = ["Elastic", "Windows", "Elastic Endgame", "Network", "Continuous Monitoring", "SecOps", "Monitoring"]
+type = "threat_match"
+
+threat_index = [ "filebeat-*"]
+threat_indicator_path = ""
+threat_language = "kuery"
+
+threat_query = """
+event.module:threatintel and
+  (threatintel.indicator.file.hash.*:* or threatintel.indicator.file.pe.imphash:* or threatintel.indicator.ip:* or
+     threatintel.indicator.registry.path:* or threatintel.indicator.url.full:*)
+"""
+
+query = """
+file.hash.*:* or file.pe.imphash:* or source.ip:* or destination.ip:* or url.full:* or registry.path:*
+"""
+
+
+[[rule.threat_filters]]
```

Comment:
I had to manually lint/format this @rw-access . There appears to be an issue with the `MarshmallowDataclassMixin.to_dict` method on `schema.dump`, but I think it is only happening on `threat_match` rules, so it may not be rendering the schema properly. As of now, it strips a lot of fields